src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/TypeScript/components/App.tsx

```diff
@@ -24,7 +24,8 @@ const useStyles = makeStyles({
     position: 'sticky',
     top: 0,
     zIndex: 1,
-    paddingBottom: '12px'
+    paddingBottom: '12px',
+    backgroundColor: tokens.colorNeutralBackground1,
```

Comment:
Ah, I had already fixed this in my PR. :smiley:

Reply:
I needed the fix for the demo in our talk - so just wanted to make sure the fix is in the checked in sources too :)